code/index.html

```diff
@@ -4,27 +4,94 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+  <body>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- 2. HERO SECTION -->
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <header class="hero">
+        <picture>
+          <source srcset="./assets/images/mobile.jpg" media="(max-width: 768px)">
+          <source srcset="./assets/images/tablet.jpg" media="(max-width: 1199px)">
+          <img class="hero-image" src="./assets/images/desktop.jpg" alt="Hero image" />
+          </picture>
+        <div class="hero-text">
+        <h1>Chick-lit Book Club</h1>
+        <h2>In Spanish!</h2>
+        <p>Improve your Spanish by chatting and having fun!</p>
+        <button class="hero-button">Sign up now</button>
       </div>
+    </header>
+
+  <section  class="forms-container">
+
+    <!-- SIGN UP FORM -->
+    <form action="http://httpbin.org/anything" method="POST">
+  
+      <label>Name  
+        <input type="text" name="message" required>
+       </label>
+       
+        <label>Email
+        <input type="email" name="email" required>
+        </label>
+        
+        <label>Spanish level
+        <select name="spanishLevel">
+          <option value="B">BEGINNER</option>
+          <option value="I">INTERMEDIATE</option>
+          <option value="A">ADVANCED</option>
+        </select>
+        </label>
+
+        <label>I accept the terms and conditions</label>
+        <input type="checkbox">
```

Comment:
If you don't nest the input inside the label, you should connect them with the attributes `for` and `id`. You can read more about this in the general feedback document.